h3/src/connection.rs

```diff
@@ -109,6 +109,12 @@ where
         settings
             .insert(SettingId::MAX_HEADER_LIST_SIZE, max_field_section_size)
             .map_err(|e| Code::H3_INTERNAL_ERROR.with_cause(e))?;
+        settings.insert(SettingId::ENABLE_CONNECT_PROTOCOL, 1)
```

Comment:
attempt to send the webtransport headers on the server side

Reply:
I would like to know the release time of the webtransport server. I am really looking forward to your project

Reply:
Hey @redoriental I encourage you to try out the h3-webtransport crate by installing it from this branch using:

h3-webtransport = { git= "https://github.com/security-union/h3.git", rev = "f4e83fb9be28ef3ea99e5feed725550f0169e5c2" } that way you wont have to wait for this PR to be merged. 

We are currently working on separating h3 and h3-webtransport, and based on the ongoing discussions, it appears that this process may take some time.

The nature of open-source development can result in a slower pace, as contributors often have limited time to dedicate, and individuals from diverse backgrounds and cultures must come to a consensus on various objectives. However, rest assured that we will ultimately deliver a well-organized and efficient webtransport server!